live-examples/js-examples/string/string-substr.html

```diff
@@ -0,0 +1,6 @@
+<pre>
+<code id="static-js">const str = 'Mozilla';
+console.log(str.substr(1,2)); // 'oz'
+console.log(str.substring(2,5)); // 'zilla'
```

Comment:
Should be `substr` not `substring`?

Reply:
Also, although it's not in the style guide :(, I think it helps readability to have a space after the comma when listing multiple parameters:

`str.substr(1, 2)`

Reply:
done